scripts/release.sh

```diff
@@ -29,9 +29,16 @@ uploadFile() {
   fi
 }
 
-echo "PULL_BASE_REF ${PULL_BASE_REF}"
+IMG=${IMG_REGISTRY}/btp-manager:$PULL_BASE_REF
+echo "Referring to image: ${IMG}"
 
-MODULE_VERSION=${PULL_BASE_REF} make module-build
+## prepare security scanning configuration for the module template
```

Comment:
where do we print sec config file content?

Reply:
cat <<EOF | tee ${FILENAME} in `create_scan_config.sh`